src/model/MultiJump.java

```diff
@@ -10,16 +11,43 @@
     private int startingPosition;
     private ArrayList<SingleJump> subJumps;
 
-    public MultiJump(int startingPosition, int endingPosition, ArrayList<SingleJump> subJumps,
+    public MultiJump(int startingPosition, int endingPosition, List<Integer> intermediatePositions,
```

Comment:
Did you change this constructor mainly for ease of creating tests? I mean it's probably more efficient for the move generator but I was just curious as to your reasoning.